### ISSUE TITLE
use the home row for moving around

### ISSUE THREAD

**Issue by kalbasit** [jooize/vim-colemak#9]
The reason why Vim moves the home row one key to the left is that on qwerty the rightmost key in the home row is the semi-column. That's not the case in Colemak though, so why not fix it?

I use Kinesis keyboard, so it makes much more sense for me to use the home row as it's intended, so a while back I forked this repo and move the keys. Is this something you'd consider?

My fork is located at https://github.com/kalbasit/vim-colemak (see README).

**Comment by jooize** [comment]
You suggest NEIO for HJKL instead of HNEI? It seems like that would make more sense, but I think I wouldn't prefer it because my accuracy with little finger is worse than with ring finger. Additionally, I prefer keeping the physical position of those actions to avoid further complicating required brain wiring.

Should we make vim-colemak mappings more flexible?

**Comment by kalbasit** [comment]
@jooize precisely. It does for sure require a different habit than the QWERTY keyboard, but isn't that the point anyway? Since we are on Colemak, we don't suffer from having the `;` under the pinkie, so I decided to move it one key to the right, and it's been working great!

I'd love to have that option so I don't have to maintain my fork. I'd love to even see it defaulting to `NEIO` so new users get used to using the right way.

**Comment by blaadje** [comment]
+1 for this

**Comment by YodaEmbedding** [comment]
HNEI makes the most sense ergonomically. The majority of the load is placed on the two strongest fingers of the hand -- and I believe they have separate tendons as well, unlike what would occur if one were to use NEIO instead.

Inter-line movement (down and up) is far more common than intra-line movement (left and right). Especially with `set relativenumber`. And we already have much more useful left/right movement operators with `w`, `b`, `e`, `f`, and `t`. The only remaining vertical keys `j` and `k` should have priority placement on the home row's stronger fingers. (In fact, I think `w` and `e`'s present placement on `y` and `u` should be swapped for a similar reason of giving greater load to the stronger fingers, but that's a different issue.)

One "compromise" is:

```
n down
e up
i left
o right
```

...but I'm not sure that it's worth the slight reduction in muscle memory.
